### ISSUE TITLE
consider, for configmap, binaryData direct support

### ISSUE THREAD

**Issue by ldemailly** [fortio/dflag#51]
It may be a bit of an issue for both the displaying of the current value (in web ui for instance) and the inability to put said data in a flag on the commandline (unless command line would use base64 for instance)

- [x] test that a confimap binaryData really ends up as binary once mounted
- [x] add binary type with base64 for cli and no base64 for cmap (well... optionally... people might still prefer not having binary)
- [x] special handling in configmap

**Comment by ldemailly** [comment]
verified cmap binaryData is indeed converted to binary
```yaml
apiVersion: v1
kind: ConfigMap
metadata:
  name: cmap
data:
  normalKey: "normal value"
binaryData:
  # $ echo -e -n '\00\01\02\03Binary\nData!' | base64
  binaryKey: "AAECA0JpbmFyeQpEYXRhIQ=="
---
apiVersion: v1
kind: Pod
metadata:
  name: debug-pod
spec:
  containers:
  - name: debug-container
    image: busybox
    command: ["sh", "-c", "sleep 3600"]
    volumeMounts:
    - name: config-volume
      mountPath: /tmp/cmap
  volumes:
  - name: config-volume
    configMap:
      name: cmap
# kubectl exec -ti debug-pod -- sh
# cd /tmp/cmap and confirm binaryKey is indeed there in binary (starts with ^@^A^B...)
```

**Comment by ldemailly** [comment]
implemented in 1.17.0
